### PR TITLE
More new sounds and fixes equip sounds

### PR DESCRIPTION
**Pull request by alsoandanswer**
- soundadd: "RFDs have new sounds, such as building, deconstructing, changing modes, etc."
  - bugfix: "RFD-S now uses the proper empty animation."
  - soundadd: "Reloading guns with magazines, bullets and shells have unique sounds."
  - soundadd: "New explosion sound has been added."
  - soundadd: "Sounds for bullets and things missing have been added."
  - soundadd: "Throwing stuff now makes a sound."
  - soundadd: "Crowbars, welders and wrenches have new sounds."
  - soundadd: "Bullet casing drop sounds should play properly now."
  - soundadd: "Equipping swords now has a cool sheathe sound."
  - soundadd: "The RPED has a new sound when in use."
  - soundadd: "New keyboard sounds."
  - soundadd: "The sound used for targeting people sounds less bitcrushed now."
  - soundadd: "New sound effects for guns jamming and being empty."
  - soundadd: "New sound effects for gauss rifles."
  - soundadd: "The sound for switching safeties on guns and laser guns have been updated."
  - soundadd: "New shotgun pump sound."
  - soundadd: "Makes revolver spin sound less crappy."
  - soundadd: "New sound for surgical drill." 
  - bugfix: "Equip sounds should play properly now."

Sounds by Chaoko99#8542 from Spriter Discord. I only know their discord tag, so.

[Here, have a snappy video.](https://youtu.be/7cn6rBOjQ3c)

Sounds are ogiginal.

### CODE REVIEW

**Comment by alsoandanswer** [comment]
!wip

**Comment by alsoandanswer** [comment]
!review

**Comment by alsoandanswer** [comment]
!bugfix

File: code/game/objects/items/weapons/RFD.dm [hunk 1]
```diff
@@ -145,135 +148,114 @@ RFD Construction-Class
...
-			build_delay = 50
-			build_cost =  5
-			build_type =  (!canRwall && W.reinf_material) ? null : "wall"
-			build_atom =  W.under_turf
-		else if(istype(T, /turf/simulated/floor))
-			build_delay = 50
-			build_cost =  10
-			build_type =  "floor"
-			build_atom =  T.baseturf
+	if(mode == 3 && istype(T,/obj/machinery/door/airlock))
```
Comment (fernerr) [inline code/game/objects/items/weapons/RFD.dm:192]: The rfd modes was moved from using defines back to numbers? I assume this is due to a faulty merge, going back to a previous version of the code.

File: code/game/turfs/simulated/wall_attacks.dm [hunk 2]
```diff
@@ -26,13 +26,13 @@
 
 /turf/simulated/wall/proc/fail_smash(var/mob/user, var/multiplier = 1)
 	user.setClickCooldown(DEFAULT_ATTACK_COOLDOWN*2.5)
-	to_chat(user, SPAN_DANGER("You smash against the wall!"))
+	to_chat(user, "<span class='danger'>You smash against the wall!</span>")
```
Comment (fernerr) [inline code/game/turfs/simulated/wall_attacks.dm:29]: Why go from using SPAN_DANGER to not? I assume it's due to a merge while resolving conflicts, I'd double check the previously conflicted files.

File: code/game/turfs/simulated/wall_attacks.dm [hunk 3]
```diff
@@ -306,7 +306,7 @@
 			if(3)
 				if (W.iscrowbar())
 					to_chat(user, SPAN_NOTICE("You struggle to pry off the cover."))
-					playsound(src, 'sound/items/Crowbar.ogg', 100, 1)
+					playsound(src, 'sound/items/crowbar.ogg', 100, 1)
```
Comment (fernerr) [inline code/game/turfs/simulated/wall_attacks.dm:309]:
```suggestion
					playsound(src, W.usesound, 100, 1)
```

File: code/game/turfs/simulated/wall_attacks.dm [hunk 4]
```diff
@@ -347,7 +347,7 @@
 			if(0)
 				if(W.iscrowbar())
 					to_chat(user, SPAN_NOTICE("You struggle to pry off the outer sheath."))
-					playsound(src, 'sound/items/Crowbar.ogg', 100, 1)
+					playsound(src, 'sound/items/crowbar.ogg', 100, 1)
```
Comment (fernerr) [inline code/game/turfs/simulated/wall_attacks.dm:350]:
```suggestion
					playsound(src, W.usesound, 100, 1)
```

**Pull request merged**